core/frontend/src/IModelConnection.ts

```diff
@@ -425,15 +425,31 @@ export abstract class IModelConnection extends IModel {
   /** Convert a point in this iModel's Spatial coordinates to a [[Cartographic]] using the Geographic location services for this IModelConnection.
    * @param spatial A point in the iModel's spatial coordinates
    * @param result If defined, use this for output
-   * @returns A Cartographic location
+   * @returns A Cartographic location (Horizontal datum depends on iModel's GCS)
    * @throws IModelError if [[isGeoLocated]] is false or point could not be converted.
    * @see [[cartographicFromSpatial]] if you have more than one point to convert, or you don't know whether the iModel has a GCS.
    */
   public async spatialToCartographicFromGcs(spatial: XYAndZ, result?: Cartographic): Promise<Cartographic> {
+    return this.toCartographicFromGcs(spatial, undefined, result);
+  }
+
+  /** Convert a point in this iModel's Spatial coordinates to a [[Cartographic]] using the Geographic location services for this IModelConnection.
+ * @param spatial A point in the iModel's spatial coordinates
+ * @param result If defined, use this for output
+ * @returns A Cartographic location (WGS84 horizontal datum)
+ * @throws IModelError if [[isGeoLocated]] is false or point could not be converted.
+ * @see [[spatialToCartographicFromEcef]]
+ * @beta
+ */
+  public async spatialToWGS84CartographicFromGcs(spatial: XYAndZ, result?: Cartographic): Promise<Cartographic> {
```

Comment:
Don't introduce new async coordinate conversion routines that take a single point.

Reply:
(Is this new convenience method really adding all that much value? User has to type "WGS84" either way...)

Reply:
We are not forced to expose a WGS84 variant, depends if we want to expose `datumOrGCRS` in our API.  Being a union type, I find it rather confusing, so I thought having a dedicated method (and its own documentation) would make it simpler.

What do you suggest?

Reply:
Makes sense - I'd expect many more users to be familiar with and/or operating within WGS84 (esp coming from Cesium) vs knowing/caring about other specific datums.